easybuild/easyconfigs/t/texinfo/texinfo-7.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,42 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+# Update: Petr Král (INUITS)
+
+easyblock = 'ConfigureMake'
+
+name = 'texinfo'
+version = '7.1'
+
+homepage = 'https://www.gnu.org/software/texinfo/'
+description = """Texinfo is the official documentation format of the GNU project."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['deeec9f19f159e046fdf8ad22231981806dac332cc372f1c763504ad82b30953']
+
+builddependencies = [('binutils', '2.40')]
+
+osdependencies = ['texlive']
+
+preinstallopts = "make TEXMF=%(installdir)s/texmf install-tex && "
+
+# This will overwrite a users $TEXMFHOME so this module is best used as a build dependency
+modextravars = {'TEXMFHOME': '%(installdir)s/texmf'}
+modloadmsg = "\n\nWARNING: This texinfo module has (re)defined the value for the environment variable $TEXMFHOME.\n"
+modloadmsg += "If you use a custom texmf directory (such as ~/texmf) you should copy files found in the\n"
+modloadmsg += "new $TEXMFHOME to your custom directory and reset the value of $TEXMFHOME to point to that space:\n"
+modloadmsg += "\tcp -r $TEXMFHOME/* /path/to/your/texmf\n"
+modloadmsg += "\texport TEXMFHOME=/path/to/your/texmf\n\n"
+
+sanity_check_paths = {
+    'files': [
+        'texmf/tex/texinfo/texinfo.tex',
+        'bin/install-info', 'bin/makeinfo', 'bin/pdftexi2dvi', 'bin/pod2texi', 'bin/texi2pdf'
+    ],
+    'dirs': ['bin', 'share', 'texmf'],
+}
```

Comment:
@PetrKralCZ can you add a few `sanity_check_commands` here?

Reply:
None of the existing `texinfo` easyconfigs have `sanity_check_commands`, so I won't block this PR over this (especially because `texinfo` is a PITA because of the `texlive` OS dependency)...